content/en/tracing/guide/ingestion_sampling_with_opentelemetry.md

```diff
@@ -43,38 +45,38 @@ With OpenTelemetry, you can configure sampling both in the OpenTelemetry librari
 
 ### SDK-level sampling
 
-At the SDK level, you can implement head-based sampling. Head-based sampling means that the sampling decision is taken at the beginning of the trace. It is particularly useful for high throughput applications, for which you know that you do not need visibility over 100% of the traffic to monitor the application health, namely to control the overhead introduced by OpenTelemetry.
+At the SDK level, you can implement _head-based sampling_, which is when the sampling decision is made at the beginning of the trace. This type of sampling is particularly useful for high throughput applications, for which you know that you do not need visibility over 100% of the traffic to monitor the application health. It can also help control the overhead introduced by OpenTelemetry.
 
-[TraceIdRatioBased][5] and [ParentBased][6] are 2 built-in samplers that allow you to implement deterministic head-based sampling based on the trace_id at the SDK level.
+[TraceIdRatioBased][5] and [ParentBased][6] are the SDK's built-in samplers that allow you to implement deterministic head-based sampling based on the `trace_id` at the SDK level.
 
-When using head-based sampling, the APM metrics will be computed **on the sampled traffic**, as only the sampled traffic is sent to the OpenTelemetry Collector / Datadog Agent.
+With head-based sampling, the APM metrics are computed **on the reduced traffic**, since only the sampled traffic is sent to the OpenTelemetry Collector or Datadog Agent, which is where the metrics calculation is done.
```

Comment:
```suggestion
With head-based sampling, the APM metrics are computed **on the sampled traffic**, since only the sampled traffic is sent to the OpenTelemetry Collector or Datadog Agent, which is where the metrics calculation is done.
```